ocean_lib/test/test_example_config.py

```diff
@@ -33,11 +33,11 @@ def test_ganache_example_config(monkeypatch):
 def test_polygon_example_config(monkeypatch):
     """Tests the config structure of Polygon network."""
 
-    monkeypatch.setenv("OCEAN_NETWORK_URL", "https://rpc.polygon.oceanprotocol.com")
+    monkeypatch.setenv("OCEAN_NETWORK_URL", "https://polygon-rpc.com")
     config = ExampleConfig.get_config()
 
     assert config.chain_id == 137
-    assert config.network_url == "https://rpc.polygon.oceanprotocol.com"
+    assert config.network_url == "https://polygon-rpc.com"
```

Comment:
The old RPC for Polygon doesn't work?

Reply:
nope, and this fix is done across all current PRs for the checks to pass